jupyterlab_caip_optimizer/setup.cfg

```diff
@@ -0,0 +1,5 @@
+[bdist_wheel]
+universal=1
+
+[metadata]
```

Comment:
This setting can be removed akin to https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/jupyterlab_gcpscheduler/setup.cfg.